README.md

```diff
@@ -10,34 +10,38 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 **Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
 
-- [Overview](#overview)
-- [Quick Start](#quick-start)
-  - [Setting up a channel to sync resourcese between your hub cluster and a object bucket](#setting-up-a-channel-to-sync-resourcese-between-your-hub-cluster-and-a-object-bucket)
-  - [Trouble shooting](#trouble-shooting)
-- [Community, discussion, contribution, and support](#community-discussion-contribution-and-support)
-- [Security Response](#security-response)
-- [References](#references)
-  - [multicloud-operators repositories](#multicloud-operators-repositories)
+- [multicloud-operators-application](#multicloud-operators-application)
+    - [What is the multicloud-operators-application](#what-is-the-multicloud-operators-application)
+    - [Getting started](#getting-started)
+    - [References](#references)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
-## Overview
 
-Channel controller sync and promote resources from the target source to a channel namespace on your hub cluster or to a host(such as object bucket). Then your subscription can consume these resources from the channel directly.
+## What is the multicloud-operators-channel
```

Comment:
Used "what is" since you use it in the other one. We can do overview across the board but trying to get them all the same, up to you...